docs/pages/reference/mesheryctl/mesheryctl-design-apply.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
all those mesheryctl-design-*.md files don't need to be created, they will be autogenerated